Automattic/woocommerce-payments
# Add footer elements to the new deposits overview card

Fixes #5719 

#### Changes proposed in this Pull Request

This PR adds the footer which contains the button (links to deposits list table), and a link to the deposit schedule settings. 

<img width="712" alt="Screenshot 2023-03-24 at 11 26 09 am" src="https://user-images.githubusercontent.com/8490476/227400822-ac74495d-b2a8-4931-ae08-f2ae570d3d4a.png">

> **Note**
> This PR doesn't include any JS tests because there's currently no test stubs set up for this card. This will be done as part of https://github.com/Automattic/woocommerce-payments/pull/5799 or a follow up PR. These elements don't have any complex states or logic that needs specific testing.

#### Testing instructions


1. Set `wcpay_feature_simplify_deposits_ui` to `1` in the wp-options table
2. From WP Admin go to Payments to view the Overview page
3. Check the Next deposits card contains a footer and the button and link send you to the correct locations.


-------------------


**Post merge**
